tests/integration/test_spans.py

```diff
@@ -1662,6 +1703,71 @@ def summarize_outcomes():
     outcomes_consumer.assert_empty()
 
 
+def test_rate_limit_is_consistent_between_transaction_and_spans(
+    mini_sentry,
+    relay_with_processing,
+    transactions_consumer,
+    spans_consumer,
+    outcomes_consumer,
+):
+    """Rate limits for indexed are enforced consistently after metrics extraction.
+
+    This test does not cover consistent enforcement of total spans.
+    """
+    relay = relay_with_processing()
+    project_id = 42
+    project_config = mini_sentry.add_full_project_config(project_id)
+    project_config["config"]["features"] = [
+        "projects:span-metrics-extraction",
+        "organizations:standalone-span-ingestion",
+        "organizations:indexed-spans-extraction",
+    ]
+    project_config["config"]["quotas"] = [
+        {
+            "categories": ["transaction"],
```

Comment:
Could you parametrize this with `@pytest.mark.parametrize("category", ["transaction", "transaction_indexed"])`.

Reply:
Might need to fake the `metrics_extracted` flag on the envelope item to see a difference.